.gitignore

```diff
@@ -0,0 +1,68 @@
+*.log
+*.swp
+*.pyc
+.DS_Store
+Thumbs.db
+node_modules
+.tmp
+.sass-cache
+bower_components
+dist
+.idea
+*.iml
+*/.sass-cache
+*/node_modules
+node/static/avatars
```

Comment:
please keep .gitignore file clean, only ignore the files that should be ignored in the project.